pkg/gatewayserver/io/ws/config.go

```diff
@@ -28,6 +28,8 @@ type Config struct {
 var DefaultConfig = Config{
 	UseTrafficTLSAddress: false,
 	WSPingInterval:       30 * time.Second,
-	TimeSyncInterval:     30 * time.Minute,
+	// Assuming 5ppm of drift, this means a drift of 5 microseconds in one second.
+	// A drift of 1 millisecond would occur every 200 seconds in such a situation.
```

Comment:
And we need 1 millisecond precision because of class B, right?

Reply:
Yes, trying to keep it as small as possible, given that [real time transfer is slightly more complicated](https://github.com/TheThingsNetwork/lorawan-stack/issues/4852).